flocx_ui/test/integration/horizon.conf

```diff
@@ -0,0 +1,4 @@
+[plugin]
+is_plugin=True
+plugin_page_path=flocx_ui.test.integration.pages
+plugin_page_structure='{"Project": {"Other": {"_": ["Flocx"]}}}'
```

Comment:
Add a new line. It is a nice practice to add '\n' which will help with diffs and EOF later.